test/generator/implementation_test.rb

```diff
@@ -12,11 +12,6 @@ def test_version
       assert_equal 1, subject.version
     end
 
-    def test_slug
-      subject = Implementation.new(paths: FixturePaths, slug: 'alpha')
-      assert_equal 'alpha', subject.slug
-    end
```

Comment:
This is not a test of core ruby functionality, it is a test that Implementation responds to slug.